vrk-kpa/suomidigi
# Added Swedish language.

TO TEST:
- make fresh
- Check that content can be created in Swedish
- Check that interface shows up in Swedish on Swedish language ( /sv/ )
